### DISCUSSION TITLE
Settings Profiles

### DISCUSSION THREAD

**Discussion by AnassAzeroual** [microsoft/vscode-discussions#214] [Extension Development QnA] [answered]
Hello everybody! Finally, settings profiles in VS Code insiders.

So now I can create two or more profiles, each profile has their extensions installed 
For example, if you develop with Angular in front-end you will create profiles for angular and then install all extensions you want and related to Angular 

It's also helping the performance and the lead of VS Code extensions 

It's an amazing idea, I can't wait to see it in vs. code

**Comment by alefragnani** [comment, marked as answer]
Hi @AnassAzeroual ,

It is already available in the stable release. You just have to enable it via the experimental setting 😬 

```json
"workbench.experimental.settingsProfiles.enabled": true
```

And yes, it is a great feature!

**Reply by AnassAzeroual** [reply]
Hi @alefragnani,

oh, great 🤩, 
Thank you so much
